webpack/assets/javascripts/react_app/redux/actions/bookmarks/bookmarks.fixtures.js

```diff
@@ -16,28 +20,37 @@ export const requestData = {
   response: { results: bookmarks },
 };
 
+const APIMiddlewareAction = {
```

Comment:
This is the bookmarks.fixtures file which in this case is saving the relevant snapshot for tests.
if you'll look down over the `onFailureActions` and `onSuccessActions` you will see it is saving kind of a snapshot of the actions that are being called. this is a fix to those tests because earlier there was no middleware chaining for the APIMiddleware.